sdk/src/store.rs

```diff
@@ -2621,7 +2708,10 @@ impl Store {
                 .await?
         };
 
-        let intermediate_output: Vec<u8> = Vec::new();
+        let intermediate_output: Vec<u8> = safe_vec(
+            stream_len(input_stream)? + MANIFEST_RESERVE_SIZE as u64,
+            None,
+        )?;
```

Comment:
What is the error here? We probably want something more specific to know if we hit this.